python-backend/app/api/documents/expected/resources/expected_document_uploads.py

```diff
@@ -98,8 +99,17 @@ def put(self, expected_document_guid):
             db.session.commit()
         else:
             return self.create_error_payload(400, 'Must specify either Mine Document GIUD or Docuemnt Manager GUID'), 400
- 
-        return expected_document.json()           
+
+        try:
+            expected_document.exp_document_status_code = 'PRE'
+            # set received_date to first upload date
+            if len(expected_document.mine_documents) == 1:
+                expected_document.received_date = datetime.now()
+            expected_document.save()
+        except AssertionError as e:
+            return self.create_error_payload(500, 'Unable to update document status'), 500
+
+        return expected_document.json()
```

Comment:
It was desired for only minespace to automatically set received date and status.